csodbpy/utils.py

```diff
@@ -0,0 +1,14 @@
+def startEngine(connectionString):
+   '''
+   Instantiates a database connection from a list of credentials
+   connectionString: dictionary of database credentials
+   returns: SQLAlchemy database engine
+   '''
+
+   from sqlalchemy import create_engine
+
+   cs = connectionString
+   
+   engine = create_engine('postgresql://' + cs['SQL_USERNAME'] + ':' + cs['SQL_PASSWORD'] + '@' + cs['SQL_HOSTNAME'] + ':' + cs['SQL_PORT'] + '/' + cs['SQL_DATABASE'])
```

Comment:
You can let sqlalchemy build the URL instead of this string addition:

```
from sqlalchemy.engine import url as dburl

dbconfig = {
    "drivername": "postgresql",
    "username": cs['SQL_USERNAME'],
    "password": cs['SQL_PASSWORD'],
    "host":cs['SQL_HOSTNAME'],
    "port": cs['SQL_DATABASE'],
    "database": cs['SQL_DATABASE']
}
engine = create_engine(dburl(**dbconfig))
```

Reply:
Even better yet, if you can make the keys in cs like above. You can just do `dburl(**cs)`